plugin.json

```diff
@@ -35,9 +35,9 @@
             },
             {
                 "key": "EnableOAuth",
-                "display_name": "Enable OAuth",
+                "display_name": "Enable OAuth (Beta)",
                 "type": "bool",
-                "help_text": "When true, OAuth will be used as authentication means with Zoom. \n Please enable only either one of OAuth based or Password base authentication.",
```

Comment:
If requiring only one of Oauth or Password based authentication, I feel we should account for that in code and not rely on the admin to make sure this happens.

Reply:
It is also taken care in the code (the plugin should not enable if both are checked). Nevertheless, it is important to make sure the admin understand why it is failing.